docs/.vuepress/public/cdr-fonts.css

```diff
@@ -0,0 +1,45 @@
+@font-face {
```

Comment:
can't this be pulled in from @rei/cedar ?

Reply:
I believe this is coming from here 
```
"scripts": {
    "dev": "npm run copy-cedar && vuepress dev docs",
    "build": "npm run copy-cedar && cross-env NODE_ENV=production vuepress build docs && npm run copy-cedar-font",
    "copy-cedar": "cp ./node_modules/@rei/cedar/dist/cedar.css ./docs/.vuepress/public/cedar.css",
    "copy-cedar-font": "cp ./node_modules/@rei/cedar/dist/cdr-fonts.css ./docs/.vuepress/public/cdr-fonts.css"
  },
```

Reply:
it should be added to the .gitignore like cedar.css